config/routes.rb

```diff
@@ -30,6 +25,13 @@
     delete 'logga_ut' => 'devise/sessions#destroy', as: :destroy_user_session
   end
 
+  # tags and exlinks
+  get 'ex_links/tags' => 'ex_links#list_tags'
+  get 'ex_links/del_unused_tags' => 'ex_links#del_unused_tags'
+  get 'ex_links/check_dead' => 'ex_links#check_dead'
+  get 'ex_links/check_expired' => 'ex_links#check_expired'
+  resources :ex_links
```

Comment:
The rule as I have been taught:

* Get never modifies any resource. Two gets is the same as just one.
* Put/patch modifies a named resource (/images/1). Two puts are the same as just one.
* Post modifies or creates a yet unnamed resource (images/). Two posts are not the same as one.
* Delete is a special put that can only be run once and has no body.